packages/core/src/mm/ovl_patch/play.c

```diff
@@ -370,7 +370,7 @@ void preInitTitleScreen(void)
 
     /* Load the entrance */
     entrance = gComboCtx.entrance;
-    if (Config_Flag(CFG_ER_ANY))
+    if (Config_Flag(CFG_ER_ANY) || (Config_Flag(CFG_OOT_START_ADULT) && (!Config_Flag(CFG_DOOR_OF_TIME_OPEN) || (Config_Flag(CFG_OOT_SWORDLESS_ADULT) && Config_Flag(CFG_OOT_TIME_TRAVEL_REQUIRES_MS))) && Config_Flag(CFG_MM_CROSS_WARP_ADULT) && (Config_Flag(CFG_MM_OWL_SHUFFLE) || Config_Flag(CFG_MM_PRE_ACTIVATED_OWL_ANY))))
```

Comment:
Why such a drastic change?

Reply:
"Since the worry is being able to have a free ticket to South Clock Town with Song of Time, this will prevent you from having that ticket in the circumstances below:"

Reply:
I didn't ask for a quote of the PR description. An actual example is preferable.

Reply:
And i provided such an example. This is currently the only other case where you could have MM access without having access to South Clock Town, until Closed Forest is a thing.